### PR TITLE
Content mui button migration

### PR DESCRIPTION
**Pull request by d88naimi**
Added a little refactored to confirm dialog for internal external link edits 

legacy 
Screen Shot 2022-05-11 at 3 34 00 PM

MUI buttons
Screen Shot 2022-05-11 at 3 34 11 PM

### CODE REVIEW

**Review by agalin920 — APPROVED** [review]
Approved

**Comment by shrunyan** [comment]
image

These changes break the collapse button layout. Please do another pass and verify all the changed elements

**Review by shrunyan — CHANGES_REQUESTED** [review]
image

These changes break the collapse layout. Please do another pass and verify all changes.

**Comment by d88naimi** [comment]
@shrunyan fixed content nav collapse button 
Screen Shot 2022-05-23 at 11 06 51 AM

File: src/apps/content-editor/src/app/views/ItemList/SetActions/SetActions.less [hunk 1]
```diff
@@ -8,7 +8,7 @@
   padding: 8px 8px 4px 8px;
 
   .I18N {
-    min-width: 250px;
+    min-width: 200px;
```
Comment (shrunyan) [inline src/apps/content-editor/src/app/views/ItemList/SetActions/SetActions.less:11]: Why did this shift 50px?

File: src/apps/content-editor/src/app/views/ItemList/SetActions/SetActions.js [hunk 2]
```diff
@@ -59,36 +59,35 @@ export class SetActions extends Component {
...
-            <Button
-              title="Create Item"
-              kind="secondary"
-              id="AddItemButton"
-              className={cx(styles.Action, styles.Create)}
-            >
-              <AppLink
-                className={styles.AppLink}
-                to={`/content/${this.props.modelZUID}/new`}
+            <AppLink to={`/content/${this.props.modelZUID}/new`}>
```
Comment (shrunyan) [inline src/apps/content-editor/src/app/views/ItemList/SetActions/SetActions.js:62]: Why did the Button > Link structure get inverted?

File: src/apps/content-editor/src/app/views/ItemEdit/Content/ActionsDrawer/ActionsDrawer.js [hunk 3]
```diff
@@ -107,22 +108,29 @@ export default function ActionsDrawer(props) {
...
-            ? styles.ActionsDrawerButtonHover
-            : ""
-        )}
+        variant="contained"
         data-cy="ActionsButton"
         title="Open for additional file information"
         onClick={() => {
           dispatch(actions.setContentActions(!ui.contentActions));
         }}
+        sx={{
```
Comment (shrunyan) [inline src/apps/content-editor/src/app/views/ItemEdit/Content/ActionsDrawer/ActionsDrawer.js:117]: Why the use of inline styles?

File: src/apps/content-editor/src/app/components/PendingEditsModal/PendingEditsModal.js [hunk 4]
```diff
@@ -39,25 +36,30 @@ export default memo(function PendingEditsModal(props) {
     };
   }, []);
 
-  const handler = (evt) => {
-    switch (evt.currentTarget.attributes["type"].value) {
+  const handler = (action) => {
+    console.log(action);
```
Comment (shrunyan) [inline src/apps/content-editor/src/app/components/PendingEditsModal/PendingEditsModal.js:40]: remove log

File: src/apps/content-editor/src/app/components/Nav/components/ContentNavToggle/ContentNavToggle.js [hunk 5]
```diff
@@ -1,37 +1,40 @@
...
-      className={cx(
-        styles.ContentNavToggle,
-        ui.contentNav ? styles.ContentNavToggleOpen : "",
-        ui.contentNavHover ? styles.ContentNavHover : ""
-      )}
       onClick={() => {
         dispatch(actions.setContentNav(!ui.contentNav));
       }}
+      fullWidth
+      sx={{
```
Comment (shrunyan) [inline src/apps/content-editor/src/app/components/Nav/components/ContentNavToggle/ContentNavToggle.js:22]: Why inline styles?

File: src/apps/content-editor/src/app/components/Nav/ContentNav.less [hunk 6]
```diff
@@ -72,7 +74,7 @@
 .NavWrap {
   white-space: normal;
   position: relative;
-  padding-bottom: 50px;
+  // padding-bottom: 50px;
```
Comment (shrunyan) [inline src/apps/content-editor/src/app/components/Nav/ContentNav.less:77]: remove

File: src/apps/content-editor/src/app/views/ItemList/SetActions/SetActions.less (hunk 1 above)
Comment (d88naimi) [reply src/apps/content-editor/src/app/views/ItemList/SetActions/SetActions.less:11]: The UI was breaking for smaller windows, I18N was taking  a lot of extra space then it needed and also shortening resolved the broken UI.

File: src/apps/content-editor/src/app/views/ItemList/SetActions/SetActions.js (hunk 2 above)
Comment (d88naimi) [reply src/apps/content-editor/src/app/views/ItemList/SetActions/SetActions.js:62]: The inverted way was messing with button styles plus this method doesn't follow our convention through out our app. We typically wrap Link > button

File: src/apps/content-editor/src/app/views/ItemEdit/Content/ActionsDrawer/ActionsDrawer.js (hunk 3 above)
Comment (d88naimi) [reply src/apps/content-editor/src/app/views/ItemEdit/Content/ActionsDrawer/ActionsDrawer.js:117]: Inline styles is the way we are moving forward with the migration as discussed. This follows our Marketing site and MUI conventions. 

CC: @agalin920

File: src/apps/content-editor/src/app/components/Nav/ContentNav.less (hunk 6 above)
Comment (d88naimi) [reply src/apps/content-editor/src/app/components/Nav/ContentNav.less:77]: Inline styles is the way we are moving forward with the migration as discussed. This follows our Marketing site and MUI conventions.

CC: @agalin920

Comment (shrunyan) [reply src/apps/content-editor/src/app/components/Nav/ContentNav.less]: please take a second look at this change request

File: src/apps/content-editor/src/app/views/ItemList/SetActions/SetActions.less (hunk 1 above)
Comment (shrunyan) [reply src/apps/content-editor/src/app/views/ItemList/SetActions/SetActions.less:11]: Was it breaking before the component swap?

File: src/apps/content-editor/src/app/components/Nav/ContentNav.less (hunk 6 above)
Comment (d88naimi) [reply src/apps/content-editor/src/app/components/Nav/ContentNav.less]: With 50px 
Screen Shot 2022-05-24 at 10 34 46 AM

Without 50p
Screen Shot 2022-05-24 at 10 35 38 AM
x

File: src/apps/content-editor/src/app/views/ItemList/SetActions/SetActions.less (hunk 1 above)
Comment (shrunyan) [reply src/apps/content-editor/src/app/views/ItemList/SetActions/SetActions.less:11]: perfect

File: src/apps/content-editor/src/app/components/Nav/ContentNav.less (hunk 6 above)
Comment (shrunyan) [reply src/apps/content-editor/src/app/components/Nav/ContentNav.less]: sorry should have been clearer. please remove the commented out code. no need to leave that lying around

Comment (shrunyan) [reply src/apps/content-editor/src/app/components/Nav/ContentNav.less]: also what changed to cause needing this style change? layout seemed to be fine before.

Comment (d88naimi) [reply src/apps/content-editor/src/app/components/Nav/ContentNav.less]: @shrunyan Yes I have reverted this one and brought back the 50px it adds some more breathing room.

Comment (shrunyan) [reply src/apps/content-editor/src/app/components/Nav/ContentNav.less]: https://github.com/zesty-io/manager-ui/pull/1053/files/ffb38ff3a8548575e10fe5b3a0fa6b595bd9ecc1#diff-88a95acfd1c51e6594b4a35ce63dbe7f05dc0b004be9745c6ccbdeadf53230efR77

still shows as commented out

Comment (d88naimi) [reply src/apps/content-editor/src/app/components/Nav/ContentNav.less]: This is looking at an outdated commit. 
https://github.com/zesty-io/manager-ui/blob/content-mui-button-migration/src/apps/content-editor/src/app/components/Nav/ContentNav.less#L77

File: src/apps/content-editor/src/app/components/Nav/ContentNav.less [hunk 7]
```diff
@@ -4,6 +4,8 @@
   overflow: hidden;
   box-shadow: none;
   height: calc(100vh - 54px);
+  display: flex;
```
[the remark this answers is not in the document]
Comment (d88naimi) [reply src/apps/content-editor/src/app/components/Nav/ContentNav.less:7]: @stuart after moving the toggle button inside content nav these styles then allowed me to push the button child to the bottom.

**Review by shrunyan — APPROVED** [review]

**Pull request merged**